src/Uno.Templates/content/unoapp/MyExtensionsApp._1/MyExtensionsApp._1.csproj

```diff
@@ -217,6 +217,8 @@
 		</When>
 		<Otherwise>
 			<ItemGroup>
+				<!-- Include all images by default - matches the __WindowsAppSdkDefaultImageIncludes property in the WindowsAppSDK -->
+				<Content Include="**/*.png;**/*.bmp;**/*.jpg;**/*.dds;**/*.tif;**/*.tga;**/*.gif"  Exclude="bin\**;obj\**" />
```

Comment:
combine with `Assets\**` otherwise we could end up adding assets twice

Reply:
done